fdbserver/storageserver.actor.cpp

```diff
@@ -1611,6 +1612,70 @@ void merge(Arena& arena,
 	}
 }
 
+ACTOR Future<Optional<Value>> quickGetValue(StorageServer* data, StringRef key, Version version) {
+	if (data->shards[key]->isReadable()) {
+		try {
+			// TODO: Use a lower level API may be better? Or tweak priorities?
```

Comment:
I also think this needs to be a range request, to get all the key-value pairs starting with the hop prefix.